fundraiser/project/fundraiser-contract/tests/functions/core/cancel_campaign.rs

```diff
@@ -135,7 +177,8 @@ mod revert {
     #[should_panic(expected = "CampaignEnded")]
     async fn when_calling_after_deadline() {
```

Comment:
For this test case I would utilize the `produce_blocks` [function](https://fuellabs.github.io/fuels-rs/v0.33.0/testing/chains.html) provided by the provider along with some variable `offset`.  Then we have
```
let deadline = block_height + offset;
create_campaign(...);
provider.produce_blocks(offset);
cancel_campaign(...);
```